### PR TITLE
INFRA-159: Dockerfile with multiple inventory profiles

### PR DESCRIPTION
**Pull request by chinzou**
a docker project to create different Bahmni server based images
https://mekomsolutions.atlassian.net/projects/INFRA/issues/INFRA-159

### CODE REVIEW

**Comment by rbuisson** [comment]
@chinzou please add in you PR a quick description and the link to the correspondig JIRA issue.

About the startup.sh script, so you did not find out why MySQL is not starting at boot?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM centos:6.9
+RUN yum update -y && yum upgrade -y
+RUN yum install -y ftp://195.220.108.108/linux/Mandriva/devel/cooker/x86_64/media/contrib/release/mx-1.4.5-1-mdv2012.0.x86_64.rpm
+RUN yum install -y python-setuptools sudo policycoreutils policycoreutils-python selinux-policy selinux-policy-targeted libselinux-utils setools openssh-server openssh-clients
+RUN yum install -y https://dl.bintray.com/bahmni/rpm/rpms/bahmni-installer-0.90-308.noarch.rpm
+ADD setup.yml /etc/bahmni-installer/setup.yml
+ADD inventories/${inventory_file} /etc/bahmni-installer/local
+ADD resources/stop_bahmni.sh /tmp
+ADD resources/startup.sh /tmp
```
Comment (rbuisson) [inline Dockerfile]: @chinzou any particular reason for this script to be named `stop_bahmni.sh` while the other is name 'startup.sh`?
For consistency, could they be named `shutdown.sh` or the other be name `start_bahmni.sh`?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+RUN bahmni -ilocal install && chmod +x /tmp/stop_bahmni.sh && chmod +x /tmp/startup.sh && /tmp/stop_bahmni.sh && rm /tmp/stop_bahmni.sh
```
Comment (rbuisson) [inline Dockerfile]: Any reason for this to be all in one line instead of multiple lines per command?

File: Dockerfile (hunk 1 above)
Comment (chinzou) [reply Dockerfile]: @rbuisson , there is no particular reason for naming the files as listed, i think we can follow the naming as you noted ( startup.sh -> start_bahmni.sh ). thank you

File: Dockerfile (hunk 2 above)
Comment (chinzou) [reply Dockerfile]: it should be in one line ( one layer ) because exactly after the installation we need to wait until the changesets are applied

**Pull request merged**